tests/validator/validator.cpp

```diff
@@ -1764,3 +1764,36 @@ TEST(Validator, unitComplexCycle)
 
     EXPECT_EQ_ERRORS(expectedErrors, v);
 }
+
+TEST(Validator, unfoundUnitsInEncapsulatedComponents)
+{
+    // This test resulted from https://github.com/cellml/libcellml/issues/430
```

Comment:
Don't need this. We can always work it out from GitHub blame, if (really) needed.